packages/flutter/lib/src/widget_previews/widget_previews.dart

```diff
@@ -10,6 +10,12 @@ import 'package:flutter/cupertino.dart' show CupertinoThemeData;
 import 'package:flutter/material.dart' show Brightness, ThemeData;
 import 'package:flutter/widgets.dart';
 
+/// Signature for callbacks that build theming data for [Preview]s.
```

Comment:
For both of these, can you [reference] the classes where they are used? :)

Reply:
They're only used as arguments to the `Preview` annotation outside of the generated widget preview scaffolding, where we generate code to invoke these function. I'll make sure to at least reference `Preview` in both typedefs.